backend/app/api/routes/assistant.py

```
@@ -0,0 +1,538 @@
+        # project_context_str = json.dumps(project_context.dict(), indent=2)
+        
+        # Get tools
+        tools = get_all_tools()
+        
+        # Prepare Claude prompt
+        prompt = f"""You are a music production assistant in the BeatGen digital audio workstation.
+The user is asking you to help edit a track. The track ID is: {request.track_id}
+
+Here's information about the current project:
+```json
+{project_context_str}
```

Ruff (0.8.2)

Undefined name `project_context_str`

(F821)